src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -635,6 +635,7 @@ private void UpdateCacheIfNeeded(long position, ITensorValueGetter[] srcTensorGe
             {
                 if (outputCache.Position != position)
                 {
+                    _parent.Session.graph.as_default();
```

Comment:
>_parent.Session.graph.as_default(); [](start = 20, length = 35)

This line is setting the graph as the default graph every time the cursor moves to the new row. We just need to do this once in the thread of the cursor. Please fix, this is call is an interop call and can reduce performance at scoring.